File: fase-5-oop/tabuada.py
print("Bem-vindo a Tabela de Multiplicação")

for i in range (1, 10):
    print(f"\nTabuada do {i}:")
    for j in range (1, 10):
        resultado = i * j
        print(f"{i} x {j} = {resultado}")
        

File: fase05-oop/src/Tabuada.java
public class Tabuada {

    public static void main(String[] args) {

        int resultado = 0;

        System.out.println("Bem-vindo à Tabuada!");

        for (int i = 1; i <= 10; i++) {
            System.out.printf("Tabuada do %d:%n", i);
            for(int j = 1; j <= 10; j++) {
                resultado = i * j;
                System.out.printf("%d x %d:%n ",i, j, resultado);
            }
        }
    }
}
